src/Altinn.AccessManagement.Persistence/DelegationMetadataRepo.cs

```diff
@@ -523,7 +523,66 @@ LatestChanges lc
                 throw;
             }
         }
-        
+
+        /// <inheritdoc />
+        public async Task<IEnumerable<InstanceDelegationChange>> GetActiveInstanceDelegations(Guid from, List<Guid> to, CancellationToken cancellationToken = default)
+        {
+            using var activity = TelemetryConfig.ActivitySource.StartActivity(ActivityKind.Client);
+
+            string query = /*strpsql*/@"
+            WITH LatestChanges AS(
+		    SELECT 
+			    MAX(instancedelegationchangeid) instancedelegationchangeid
+		    FROM
+			    delegation.instancedelegationchanges
+		    WHERE
+			    fromuuid = @fromUuid
+                AND touuid = ANY(@toUuids)
+		    GROUP BY
+			    instancedelegationmode
+			    ,resourceid
+			    ,instanceid
+			    ,fromuuid
+			    ,fromtype
+			    ,touuid
+			    ,totype)
```

Comment:
Var det ikke her man landet på at man måtte slå de sammen til en string?

Reply:
Altså uid og type

Reply:
Nei, det blir vel værre å slå opp om vi tvinger oppslaget til å måtte ha ett forhold til type.
Har du ett sett med Uuids man skal slå opp fra/til så er det en fordel å kunne slå opp på denne uten å måtte spesifisere type. Som er trygt så lenge vi kan garantere at en ID ikke kan gjenta seg på tvers av typer.

Reply:
Men jeg innser at spørringen mangler filter på resourceId som må på plass

Reply:
Samtidig er det også en fordel å ha type i egen kolonne for å kunne gjøre oppslag som: Hent alle delegeringer fra en gitt avgiver til alle systembrukere.